server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -367,11 +368,16 @@ private DivTag renderCardApplications(AccountModel account) {
       return div();
     }
 
-    ImmutableList<ApplicationModel> newestApplicantApplications =
-        newestApplicant.get().getApplications();
-    int applicationCount = newestApplicantApplications.size();
+    ImmutableList<ApplicationModel> submittedApplications =
+        newestApplicant.get().getApplications().stream()
+            .filter(
+                application ->
+                    application.getSubmitTime() != null
+                        && application.getSubmitTime().compareTo(Instant.now()) <= 0)
```

Comment:
**suggestion:** I think it would be helpful to add a comment here explaining this code. If an application isn't submitted, is the submit time another instant?

Not sure if this is better, but another option could be using the lifecycle stage, which may be a little clearer. Seems like any submitted application would have a lifecycle stage as active https://github.com/civiform/civiform/blob/63d2163c3b2019804751bda444d1d63676ceffd2/server/app/repository/ApplicationRepository.java#L74-L163

Reply:
Great idea to use the `LifecycleStage.ACTIVE`.  I've switched to using that.